tests/topk_varlen/test_topk_varlen.py

```
@@ -0,0 +1,413 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+"""Correctness tests for flashinfer.top_k_decode.
+
+On Blackwell (sm_100+) with ``pre_idx`` supplied the GVR fast path runs.
+On other hardware the radix fallback is used; most tests still execute.
+
+Test matrix
+-----------
+test_basic_decode             — dtype × top_k × N × batch; works on all GPUs
```

Ruff (0.15.20)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)